app/bundles/LeadBundle/EventListener/DoNotContactSubscriber.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
```

Comment:
The newly added files should have the current year here. Do not change it for existing files you modify. Just the newly added.